src/wasm2asm.h

```diff
@@ -109,6 +109,7 @@ class Wasm2AsmBuilder {
 
 public:
   struct Flags {
+    Flags() : debug(false), pedantic(false) {}
     bool debug: 1;
```

Comment:
Not that anyone asked, but IMO this shouldn't be using bitfields either. :-)

Reply:
I agree.

Also personally I'd prefer `bool debug = false;` (init in the declaration). But lgtm either way.

Reply:
What?  You want to waste almost 8 entire bytes for every command line run!  You know thats almost 64 bits right?